scripts/links.js

```diff
@@ -0,0 +1,41 @@
+import jQuery from 'jquery';
+import { jsdom } from 'jsdom';
+import path from 'path';
+import replaceExtension from 'replace-ext';
+import isAbsoluteUrl from 'is-absolute-url';
+
+const $ = jQuery(jsdom().defaultView);
+
+/**
+ * Markdown doesn't render absolute links to markdown files (see https://github.com/mkdocs/mkdocs/issues/1172)
+ * 
+ * @param {String} html 
+ */
+export function reassignMarkdownLink (html) {
+  const jqHTML = $(html);
+  const anchorTags = jqHTML.find('a');
+  anchorTags.each((index, tag) => {
+    const anchorTag = $(tag);
+    const href = anchorTag.attr('href');
+
+    if (href && !isAbsoluteUrl(href)) {
+      const ext = path.extname(href);
+      if (ext === '.md') {
+        anchorTag.attr('href', `${replaceExtension(href, '')}/index.html`);
+      }
+    }
+  });
+  return jqHTML.html();
+}
+
+/**
+ * Takes a document, parses out the body and then adds fenced code to the body
+ * @param {*} htmlDocString 
+ */
+export function reassignMarkdownLinkDocument (htmlDocString) {
+  const body = '<div id="body-mock">' + htmlDocString.replace(/^[\s\S]*<body.*?>|<\/body>[\s\S]*$/ig, '') + '</div>';
```

Comment:
Wouldn't it be more precise to do the replacement using jQuery?

```
$("body").html("new content");
```

Reply:
jQuery unfortunately doesn't parse documents

Reply:
[why not](https://stackoverflow.com/questions/40790466/how-to-read-a-html-file-for-jquery-using-node-js)?

Reply:
Hmmm, I'll try that way and make a new PR if it works.

Reply:
I just tried this:

```javascript
export function reassignMarkdownLinkDocument (htmlDocString) {
  const $html = $(htmlDocString);
  const $body = $html.find('body');
  console.log('!!!!!', $body.html());
```

And it logs 'undefined'.

Reply:
thanks for trying this. Just to make sure - what is the output of $html.html() ?